h2o-py/h2o/estimators/rulefit.py

```diff
@@ -23,6 +24,7 @@ class H2ORuleFitEstimator(H2OEstimator):
     algo = "rulefit"
     supervised_learning = True
 
+    @deprecated_params({'Lambda': 'lambda_'})
```

Comment:
Why do you use underscore at the end of the parameter?

Reply:
because lambda is a reserved word. It is used like this also in glm (this isa copy paste from glm)

Reply:
Ok, sure.